@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -59,8 +59,8 @@ describe("MapView", function() {
         addEventListenerSpy = sinon.stub();
         removeEventListenerSpy = sinon.stub();
         canvas = ({
-            clientWidth: 0,
-            clientHeight: 0,
+            clientWidth: 400,
```

Comment:
Should be big enough to provide tested precision: probably 1920x1080 as in other tests.